plugins/woocommerce-admin/client/marketplace/components/product-card/product-card.scss

```diff
@@ -0,0 +1,145 @@
+@import '../../stylesheets/_variables.scss';
+
+.woocommerce-marketplace {
+
+	&__product-card {
+		display: flex;
+		flex-direction: column;
+		gap: $medium-gap;
+		padding: $large-gap;
+		border-radius: $grid-unit-05;
```

Comment:
This is being overridden by the product card default styling and so the card only ends up with a 1px border-radius.